assignment.md

```diff
@@ -1,23 +1,31 @@
-What is the difference between push, pull, and fetch?
+# Important Git commands to understand 
 
-# Understanding git commands push, pull, and fetch
+To assist you in understanding Git concepts and related commands, Git command definitions are provided below.
 
-**`git push`** - pushes changes from your local branch to a remote repository.
+**`git push`** - Pushes changes from your local branch to a remote repository.
 
-**`git fetch`** - fetches changes from a remote repository into the tracking branch.
+**`git fetch`** - Fetches changes from a remote repository into the tracking branch.
 
-**`git pull`** - pulls the changes from a remote branch into the tracking branch and merges the changes into your local branch. 
+**`git pull`** - Pulls changes from a remote branch into the tracking branch and merges the changes into your local branch. 
 
-Often git push and git pull are described as equivalent. This isn't entirely correct, since under the hood git pull does two things. 
-git push takes our current branch, and checks to see whether or not there is a tracking branch for a remote repository connected to it. 
-If so, our changes are taken from our branch and pushed to the remote branch. This is how code is shared with a remote repository, 
-you can think of it as "make the remote branch resemble my local branch". 
+**`git merge`** - Integrates and merges changes from another branch.  
```

Comment:
I moved and split this section.  Not sure why pull and push are stated as potentially equivalent when they are opposites in reality.  

Removed the term under the hood as this term is not universally understood.